docs/tutorial.rst

```diff
@@ -1484,3 +1484,107 @@ Running this code we get::
     -5.665181028073889
     -7.087195080578711
 
+.. _sec_tutorial_multiple_mergers:
+
++++++++++++++++++++++++++++
+Multiple merger coalescents
++++++++++++++++++++++++++++
+
+``msprime`` can simulate several multiple merger coalescent models,
+in which any number of lineages can coalesce in up to four simultaneous
+mergers. These can often be appropriate models when the distribution of
+offspring numbers among individuals in the population is highly skewed.
+Provided are the classes of Beta-Xi-coalescents and Dirac-Xi-coalescents,
+whose use is illustrated below, first for the Beta-Xi-coalescent.
+
+.. code-block:: python
+
+    def beta_multiple_merger_example():
+        ts = msprime.simulate(
+            sample_size=10,
+            recombination_rate=350,
```

Comment:
Let's not bother with recombination for the initial examples, it's just obscuring the important things.